### ISSUE TITLE
Schema feedback

### ISSUE THREAD

**Issue by wolverdude** [dcporter44/soundify-app#2]
* No song location for comments?
* You should have a like model, not just like counts, otherwise you can't prevent anyone from liking a song again and again.
* `audio_link` should be `audio_url`, likewise with `album_art_url`.

**Comment by dcporter44** [comment]
Yeah, I didn't think of that, thanks. What do you mean by "song location"?

**Comment by wolverdude** [comment]
On Soundcloud, comments are associated with a certain location on the song track, that way you can comment on a certain part of a song, not just the whole thing.
